content/blog/2021-08-31-aug-community-gems.md

```diff
@@ -0,0 +1,130 @@
+  file handling and more.
+picture: 2021-08-31/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/august-21-community-gems/45342t5t4egrf4w
+tags:
+  - Community
+  - Plots
+  - Pipelines
+  - CML
+  - Git
+---
+
+### [Q: Are TOML files supported for storing model metrics and displaying them via `dvc metrics show`?](https://discord.com/channels/485586884165107732/485596304961962003/865974923079319563)
+
+Thanks for the question @naeljaneLiblikas!
+
+DVC does not support TOML files for metrics. TOML files are only used for
+parameters.
+
+We do have an [open issue](https://github.com/iterative/dvc/issues/6402) for
+this so you can keep up with any updates.
+
+### [Q: How can I use object storage as a data source for MLOps with DVC?](https://discord.com/channels/485586884165107732/485596304961962003/866393535296176148)
+
+You have a few options with DVC for this.
+
+- Set up an external cache on your object storage bucket.
+  [Check out this doc to learn more](https://dvc.org/doc/user-guide/managing-external-data#setting-up-an-external-cache)!
+- Track a large dataset in an external location to download later.
+  [This doc will tell you more](https://dvc.org/doc/command-reference/add#example-transfer-to-remote-storage).
+
+Great question @Ravi Kumar!
+
+### [Q: Is there a recommended way to specify multiple pipelines in DVC?](https://discord.com/channels/485586884165107732/485596304961962003/864230750325047316)
+
+You'll want to keep each pipeline in a separate `dvc.yaml` if you want to work
+with multiple pipelines. Here's a bit of explanation:
+
+- Splitting a `dvc.yaml` file into multiple files is encouraged where there are
+  clear logical groupings between stages. It avoids confusion, improves
+  readability, and shortens commands by avoiding long paths preceding every
+  filename.
+- `dvc.yaml` files can be in any sub-directory or nested sub-directory in the
+  project structure and DVC will find them.
+- DVC will process them just the same as if they were one DVC file i.e.
+  dependencies between stages in different `dvc.yaml` files are still respected.
+- Each `dvc.yaml` file will have its own `dvc.lock` file in the same directory.
+
+If you want to see the rest of the explanation,
+[check out this user guide PR we have up](https://github.com/iterative/dvc.org/issues/2494).
+Please feel free to add a comment or emoji on this PR so we know how to
+prioritize this content for you!
+
+Thanks @Tups!
+
+### [Q: Is there way to allow different pipelines to have common dependencies and outputs in DVC pipelines?](https://discord.com/channels/485586884165107732/563406153334128681/867747202306146335)
+
+Good question @vgodie!
+
+It is possible to have overlapping dependencies, but not overlapping outputs.
+Having overlapping outputs introduces uncertainty into DVC commands, like
+`dvc checkout`.
+
+Sometimes people want to have overlapping directory outputs (different stages
```

Comment:
Could we provide an example? (for the sake of having some code blocks and/or images in this post - it looks a bit too "empty" at the moment?

Reply:
I'm not quite sure what to put here as a code block, but the new Q/A has a Python code block. Let me if you think that looks good enough. If not, I'll add some code in here somewhere. 🙂